src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -125,6 +126,8 @@ bool ngraph::pass::CommonOptimizations::run_on_model(const std::shared_ptr<ngrap
     common_fusions->add_matcher<ngraph::pass::InterpolateSequenceFusion>();
     common_fusions->set_name("ngraph::pass::CommonFusions");
 
+    manager.register_pass<ngraph::pass::ConcatReduceFusion>();
```

Comment:
If MOCTransformations pipeline already contains this pass so we don't have to add it into CommonOptimizations pipeline.

Reply:
I removed this transformation from MOC pipeline because we have to disable this transformation in the GNA Plugin.